CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # OSIM Changelog
 
+## [2024.6.2]
```

Comment:
Doesn't have sense to use `unreleased` until we have clear the new release to be made, so we will get the specific items we need from unreleased and move it to the new release block?

Reply:
Yes--I think I misunderstood the last team conversation about release documenting, thinking what we were doing for a particular release was an overall change in process to move away from using an unreleased heading.